wheel/src/run_program.rs

```diff
@@ -52,12 +47,13 @@ pub fn deserialize_and_run_program2(
     opcode_lookup_by_name: HashMap<String, Vec<u8>>,
     max_cost: Cost,
     flags: u32,
-) -> PyResult<(Cost, LazyNode)> {
-    let mut allocator = if flags & LIMIT_HEAP != 0 {
-        Allocator::new_limited(500000000, 62500000, 62500000)
-    } else {
-        Allocator::new()
-    };
+) -> PyResult<(PyObject, LazyNode)> {
+  let mut allocator = if flags & LIMIT_HEAP != 0 {
+      Allocator::new_limited(500000000, 62500000, 62500000)
+  } else {
+      Allocator::new()
+  };
```

Comment:
something happened to the indentation here

Reply:
Something weird is going on with `cargo fmt`. I'm investigating.